packages/components/src/components/ProgressTracker/ProgressTrackerItem.tsx

```diff
@@ -269,6 +276,8 @@ export const ProgressTrackerItem = (props: ProgressTrackerItemProps) => {
         onClick={
           !disabled && handleStepChange
             ? () => handleStepChange(index)
+            : onClick
+            ? () => onClick(index)
             : undefined
         }
```

Comment:
Her er vel `handleStepChange` prioritert over `onClick`, men jeg tenker nok at det burde være motsatt, `onChange` kan brukes til å overstyre funksjonaliteten for visse steg tenker jeg.

Skriv gjerne dette i JSDoc for `onClick` også 😇

Reply:
Hmm, godt poeng.. men mulig `onClick` ikke burde overstyre, men heller kalle begge isåfall? Kan wrappe hele greia i en funksjon som kaller en, ingen eller begge avhengig av hvilke funksjoner som er spesifisert i props.

Reply:
Lar `onClick` overstyre `handleStepChange`